sdcm/utils/health_checker.py

```diff
@@ -38,10 +38,10 @@ def check_nodes_status(nodes_status: dict, current_node, removed_nodes_list=())
     if not nodes_status:
         LOGGER.warning("Node status info is not available. Search for the warning above")
         return
-    LOGGER.info("Status for %s node %s", node_type, current_node.name)
+    LOGGER.debug("Status for %s node %s", node_type, current_node.name)
```

Comment:
I think that it helps to have these messages as info (when looking into Jenkins output).
don't you think?

Reply:
this print looks like: 
```Status for regular node rolling-upgrade-4-5-ubuntu-focal-db-node-d6097fd3-0-2``` it doesn't tell us anything importent.

if there a failure in the health checker, it would raise an error or critical event which would be seen on the stdout.